src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -58,18 +66,48 @@ private void configureGameBehavior() {
     }
   }
 
+  /** Creates an input stream for a joystick. */
+  private InputStream createJoystickStream(InputStream input, double maxSpeed, double maxRate) {
+    return input
+        .clamp(1)
```

Comment:
is this necessary? Like how would it ever give a value with magnitude greater than 1?